recipes/eta/meta.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: eta
+  version: "0.9.7"
+
+source:
+  fn: eta-0.9.7.tar.gz
+  url: https://pypi.python.org/packages/56/f4/3def7582d454d511a0b48e5eabdfd1319ccb6b5fafc810fea7dee4ca6c4d/eta-0.9.7.tar.gz
+  md5: fd6d815b2a3276ca07655c4037472576
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - eta = eta:main
+    #
+    # Would create an entry point called eta that calls eta.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - eta
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/mbreese/eta/
+  license: UNKNOWN
```

Comment:
Adding a licence would be good. Having no license means essentially we are not allowed to distribute the package.